rustls/Cargo.toml

```diff
@@ -48,9 +48,8 @@ log = { workspace = true, optional = true }
 # in the error result from `crypto::CryptoProvider::set_default()`, which does not seem
 # worth using the `critical-section` feature with its no-std installation requirements.
 # TODO: specify `once_cell` version in single place - as RECENTLY UPDATED in upstream RUSTLS
-# (It may be best to specify the `once_cell` features here, as may be needed for updates moving forward.)
-once_cell = { version = "1.20.3", default-features = false, features = ["alloc", "race"] }
-# TODO: KEEP `portable-atomic-util` OPTIONAL WITH FEATURE CONFIG IN THIS FORK
```

Comment:
need to keep this TODO comment for now ... other PR #45 with new "portable-atomic" feature name could help resolve this TODO item